extract_function_sigs.py

```diff
@@ -8,7 +8,9 @@
 import os.path
 import sys
 import contextlib
-
+from lxml import html
```

Comment:
I don't think we need to make such a simple script have a dependency (outside of the python standard library)

Reply:
> A couple notes.
> 
> I also don't think the version is used for anything other than the file name, correct? If so I think it's reasonable (maybe even better) to tag it with the current git hash or just `develop`. For example, if you run the script against master, functions may have been added since the last push online.

It's just the name. Yea, the current git hash would be good.